pkg/webhook/create.go

```diff
@@ -59,26 +60,31 @@ func SetupWebhooks(ctx context.Context, cfg *rest.Config) error {
 		return err
 	}
 
+	var secretName string
 	if config.ControllerCfg.IsOpenShift() {
+		secretName = server.WebhookServerTLSSecretName
 		// Set up the certs for OpenShift
 		log.Info("Setting up the OpenShift webhook server secure service")
-		err := webhook_openshift.SetupSecureService(client, ctx, namespace)
+		err := webhook_openshift.SetupSecureService(client, ctx, secretName, namespace)
 		if err != nil {
 			return err
 		}
 	} else {
+		secretName, err := config.GetWebhooksSecretName()
+		if err != nil {
+			return fmt.Errorf("failed to create webhooks deployment: %w", err)
```

Comment:
Is it an error message copied from somewhere?
I mean we haven't tried to create deployment yet. Maybe instead - `failed to deploy webhooks due:` or something like that

Reply:
Updated the complete message -- it should now be `could not deploy webhooks server: environment variable WEBHOOK_SECRET_NAME is unset`